app/assets/stylesheets/application.scss

```diff
@@ -39,4 +39,12 @@
 
 .location {
   margin-top: 14px;
+  color: darkgrey;
+  font-size: 20px;
+  margin-left: -100px;
+  margin-right: 80px;
+}
+
+.location-dropdown {
+  margin-top: 14px;
```

Comment:
Move the css to separate file. Use this manifest file to require other stylesheets.